### PR TITLE
Steph wed docs

### PR DESCRIPTION
**Pull request by scoyle391**

### CODE REVIEW

File: reference/idp-support/auth0.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ With Auth0 as your IdP, you will need to create an `Application` to handle authe
 
   ![](../../../doc-images/scopes.png)
   
-4. In your newly created application, click on the Settings tab, add the Domain and Callback URLs for your service and ensure the "Token Endpoint Authentication Method" is set to `Post`. The default YAML installation of Ambassador Edge Stack uses `/callback` for the URL, so the values should be the domain name that points to Ambassador, e.g., `example.com/callback` and `example.com`.
+4. In your newly created application, click on the Settings tab, add the Domain and Callback URLs for your service and ensure the "Token Endpoint Authentication Method" is set to `Post`. The default YAML installation of Ambassador Edge Stack uses `/c.ambassador/oauth2/redirection-endpoint` for the URL, so the values should be the domain name that points to Ambassador, e.g., `example.com/.ambassador/oauth2/redirection-endpoint` and `example.com`.
```
Comment (LukeShu) [inline reference/idp-support/auth0.md]: Typo here. `/c.ambassador/` should be `/.ambassador/`

File: reference/idp-support/onelogin.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+3. Give your connector a name.
+4. Select the `OpenID Connect` option as your "Sign on method."
+5. Use `http(s)://{{AMBASSADOR_URL/.ambassador/oauth2/redirection-endpoint` as the value for "Redirect URI."
+6. Optionally provide a login URL.
+7. Click the **Save** button to create the connector. You will see a confirmation message.
+8. In the "More Actions" tab, select **Add App to Connector**.
+9. Select the connector you just created.
+10. Click the **Save** button.  
+
+You will see a success banner, which also brings you back to the main portal page. OneLogin is now configured to function as a OIDC backend for authentication with Ambassador.
```
Comment (LukeShu) [inline reference/idp-support/onelogin.md]: "a OIDC" → "an OIDC"

File: reference/idp-support/onelogin.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        authorizationURL: https://{{subdomain}}.onelogin.com/oidc
+        # The clientURL is the scheme and Host of your Ambassador endpoint
+        clientURL: httpi(s)://{{AMBASSADOR_URL}}
+        clientID: {{Client ID}}
+        secret: {{Client Secret}}
+```
+
+2. Next, create a [FilterPolicy](../../reference/filter-reference#filterpolicy-definition) to use the `Filter` you just created.
+
+Here is an example YAML:
```
Comment (LukeShu) [inline reference/idp-support/onelogin.md]: "an example YAML" sounds weird to me.  I wouldn't say that YAML is a countable noun.  Perhaps "some example YAML".

File: reference/idp-support/auth0.md (hunk 1 above)
Comment (scoyle391) [reply reference/idp-support/auth0.md]: changed

File: reference/idp-support/onelogin.md (hunk 2 above)
Comment (scoyle391) [reply reference/idp-support/onelogin.md]: changed

File: reference/idp-support/onelogin.md (hunk 3 above)
Comment (scoyle391) [reply reference/idp-support/onelogin.md]: updated

**Comment by nbkrause** [comment]
auth stuff looks good to me

**Pull request merged**